docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
-        nodeSelector:
-          beta.kubernetes.io/os: windows
-  ```
-
-5. Type `kubectl get pods -w` to watch the deployment of the service that takes about 30 seconds.  Once running, type `kubectl get svc` and curl the 10.x address to see the output, eg. `curl 10.244.1.4`
-
-6. Type `kubectl get svc -w` to watch the addition of the external IP address that will take about 2-5 minutes.  Once there, you can take the external IP and view in your web browser.
-
-## Example using Azure Files and Azure Disks
-### Create Azure File workload
-This example is modified after https://github.com/andyzhangx/Demo/tree/master/windows/azurefile/rs3
-
-#### 1. Create an azure file storage class
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/storageclass-azurefile.yaml```
-
-#### make sure storageclass is created successfully
-```
-kubectl get storageclass/azurefile -o wide
+```powershell
+dir ~\.ssh\id_rsa.pub
+dir : Cannot find path 'C:\Users\patrick\.ssh\id_rsa.pub' because it does not exist.
 ```
 
-#### 2. Create a pvc for azure file
-```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azurefile/pvc-azurefile.yaml```
-
-#### make sure pvc is created successfully
-```
-kubectl get pvc/pvc-azurefile -o wide
+If it does not exist, then run `ssh-keygen.exe`. Use the default file, and enter a passphrase if you wish to protect it. Be sure not to use a SSH key with blank passphrase in production.
+
+```powershell
+PS C:\Users\patrick\acs-engine> ssh-keygen.exe
+Generating public/private rsa key pair.
+Enter file in which to save the key (C:\Users\patrick/.ssh/id_rsa):
+Created directory 'C:\Users\patrick/.ssh'.
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in C:\Users\patrick/.ssh/id_rsa.
+Your public key has been saved in C:\Users\patrick/.ssh/id_rsa.pub.
+The key fingerprint is:
+SHA256:... patrick@plang-g1
+The key's randomart image is:
++---[RSA 2048]----+
+...
++----[SHA256]-----+
 ```

```

Comment:
Again just specify differences on *nix systems (i.e. no need for .exe extension, update correct paths, examples, etc.)